index.js

```diff
@@ -96,89 +103,72 @@ function myMap(array, callback) {
   }
   return result
 }
-
 // ***** Callbacks - Question 1 *****
-
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
-
-// console.log(myMap([1,2,3,4,5], triple)) 
+console.log("%cCallbacks - Question 1", "color: red")
+const triple = (arr) => {
+  return arr * 3;
+}
+console.log(myMap([1,2,3,4,5], triple))
 // // => [3,6,9,12,15]
-
-// console.log(myMap([2,4,6,8], triple))   
+console.log(myMap([2,4,6,8], triple))
 // // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
-
-
+console.log("%c----------", "color: red")
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
-
-// console.log(myMap(["Raffy", "Chase"], greet)) 
+console.log("%cCallbacks - Question 2", "color: red")
+const greet = (arr) => {
+  return `Heyyy, ${arr}`
+}
+console.log(myMap(["Raffy", "Chase"], greet))
```

Comment:
Here is my solution, first define greet, which you have,

```
const greet = name => {
  return `Hi, ${name}!`;
}
```

or simplify to 

`const greet = name => `Hi, ${name}!`;`

then define the myMap function

```
const myMap = (arr, fn) => {
  return arr.map(name => {
    return fn(name);
  })
}
```

or simplified

`const myMap = (arr, fn) => arr.map(name => fn(name));`

we are creating a new array using map, where each element is altered by applying the callback function.